tests/video/test_loading_cursor.py

```diff
@@ -0,0 +1,27 @@
+from panda3d.core import MovieVideo
+from panda3d.core import MovieVideoCursor
+import pytest
+
+def get_movie():
+    movie_path = "../samples/media-player/PandaSneezes.ogv"
+    getFile = MovieVideo.get(movie_path)
+    return getFile #Obtains a MovieVideo that references a file
+
+@pytest.fixture
+def cursor_check():
```

Comment:
This should `return` or `yield` a value to be usable as a fixture. As written, I am fairly certain it returns `None`.